es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -15,100 +15,111 @@
                 </div>
             </div>
         </div>  <!-- Make an impact -->
-        <div class="container text-gray font-size-75 pt-200 pb-300">
-            <div class="row">
-                <!-- Link columns -->
-                <div class="col col-12 col-lg-8 order-lg-1">
-                    <div class="row">
-                        <div
-                            v-for="column in content.columns"
-                            :key="column.title"
-                            class="col col-6 col-md-3 mb-150">
-                            <ul class="list-unstyled">
-                                <li class="font-weight-bolder font-size-100 mb-50">
-                                    {{ column.title }}
-                                </li>
-                                <li
-                                    v-for="link in column.links"
-                                    :key="link.text"
-                                    class="mt-25 mb-50">
-                                    <es-nav-bar-link
-                                        :href="link.url"
-                                        :target="link.newTab ? '_blank' : null"
-                                        class="text-reset">
-                                        {{ link.text }}
-                                    </es-nav-bar-link>
-                                </li>
-                            </ul>
+        <div class="bg-blue-800 py-200 px-200">
+            <div class="container px-0">
+                <div class="row">
+                    <!-- Link columns -->
+                    <div class="col col-12 col-lg-8 order-lg-1 text-white">
+                        <div class="row">
+                            <div
+                                v-for="column in content.columns"
+                                :key="column.title"
+                                class="col col-6 col-md-3 mb-150">
```

Comment:
I think the `mb-150` here may need to become `mb-150 mb-lg-0`, because it's causing too much space below the links on desktop, more than the design calls for.

There are probably a few more things you'll need to tweak in order to correct that spacing, as a number of things are contributing to it - I think it's 16px from the `<ul>` (could put a `mb-lg-0` on those) plus 24px from this `mb-150` (which the above change will solve) plus 32px from the `py-200` on the container `<div>`, adding up to a total 72px, when the design calls for 32px total.

<img width="1278" alt="Screen Shot 2024-05-08 at 12 40 46 PM" src="https://github.com/EnergySage/es-ds/assets/1350363/7f2bd1ed-a4fd-4754-a05b-aabbc44886ef">